src/ansys/fluent/core/meshing/faulttolerant.py

```diff
@@ -16,19 +13,10 @@ def fault_tolerant_workflow(**launch_args) -> EnhancedMeshingWorkflow:
 
```

Comment:
Can we pass just the meshing session to this function?

`launch_args` can be complex and it is better to handle that complexity in a single place, i.e. in `launch_fluent()` method. If we allow all laucher args in this function, we need to document them here also as this is a user-facing function (and also about the arg vals which are not supported in meshing mode).

IMHO, the wrapper should not be responsible for managing the session instance, i.e. if user needs to use the session instance while or after working with the wrapper, is there a way to do that?

Reply:
I think this makes sense. I have removed the launch part from here. I don't remember exactly why it was added here, @seanpearsonuk, was there a valid reason that we are missing now?

Reply:
@prmukherj we should make the changes suggested by @mkundu1.

Reply:
Already pushed @seanpearsonuk

Reply:
Regarding the renaming I was awaiting your feedback. Will do it soon. Thank you.